package.json

```diff
@@ -38,15 +38,16 @@
     "@babel/preset-env": "7.16.11",
     "@babel/preset-typescript": "^7.16.7",
     "@types/jest": "^27.4.1",
-    "@types/react": "^17.0.39",
+    "@types/react": "^18.0.5",
     "@types/react-native": "^0.66.16",
     "babel-jest": "^27.5.1",
     "babel-plugin-module-resolver": "^4.1.0",
     "babel-preset-react-native": "5.0.2",
     "jest": "^27.5.1",
     "lerna": "^4.0.0",
+    "metro-react-native-babel-preset": "^0.70.1",
```

Comment:
Can we move this metro dependency to the sample app level as it should only be used by the sample app?